cpp/src/arrow/compute/kernels/scalar_arithmetic.cc

```diff
@@ -1667,6 +1930,22 @@ const FunctionDoc trunc_doc{
     ("Calculate the nearest integer not greater in magnitude than to the "
      "argument element-wise."),
     {"x"}};
+
+const FunctionDoc round_doc{
+    "Round the arguments to a given precision element-wise",
```

Comment:
I don't think "element-wise" is informational (what would a non-element-wise round function?).

Reply:
So just "Round to a given precision" perhaps?